lib/osf-components/addon/components/contributors/widget/template.hbs

```diff
@@ -8,6 +8,9 @@
         <label for={{currentContributorFieldId}}>
             {{t 'osf-components.contributors.currentContributors'}}
         </label>
+        <div>
```

Comment:
should this use `local-class='warning-container'`? and should warning-container use `color: $brand-danger`?

Reply:
Oh, you beat me to it!